lib/proxy_api/resource.rb

```diff
@@ -84,5 +84,11 @@ def put(payload, path = "")
     def delete(path)
       resource[path].delete
     end
+
+    def cached_get(path = nil, payload = {})
+      Rails.cache.fetch("proxies/#{url}", :expires_in => 3.minutes) do
+        get(path, payload)
```

Comment:
do we store somewhere when was the data last refreshed? (we could use that to display in the UI)